pyproject.toml

```diff
@@ -235,12 +235,8 @@ ignore = [
     "S324", # Probable use of insecure hash functions in `hashlib`
     "S506", # Probable use of unsafe loader `BaseLoader` with `yaml.load`. Allows instantiation of arbitrary "object"s. Consider `yaml.safe_load`.
     "SIM102", # Use a single `if` statement instead of nested `if` statements
-    "SIM108", # Use ternary operator instead of `if`-`else`-block
-    "SIM112", # Use capitalized environment variable `HTTPS_PROXY` instead of `https_proxy`
     "SIM114", # Combine `if` branches using logical `or` operator
-    "SIM117", # Use a single `with` statement with multiple contexts instead of nested `with` statements
```

Comment:
I like this in theory, but in practice the resulting code here feels a bit weird to me. idk wdyt?